components/builder-web/README.md

```diff
@@ -4,40 +4,36 @@ This is the web application for Builder. It's a single-page application built wi
-    environment: "production",
-    github_client_id: "0c2f738a7d0bd300de10",
-    source_code_url: "https://github.com/habitat-sh/habitat",
-    tutorials_url: "https://www.habitat.sh/learn",
-    www_url: "https://www.habitat.sh",
-    ...
-});
-```
-### Running the Builder API Service
+Therefore, these steps are part of the setup of the full dev environment that is outlined in the overall [Builder  Development](../../DEVELOPING.md) doc.
+
+### Prerequisites
+
+You should have gone through the steps in the Builder Development document to set up and configure the Builder API service, and created your Github app.
 
-While it's possible to run this application without a concurrently running Builder API service, you won't be able to perform the kinds of actions that rely on that API (like create an origin, list and browse packages, sign in and out, and so on). To stand up a Builder API service locally, see the [BUILDER_DEV](../../BUILDER_DEV.md) doc.
+You will need the Github app id and client id for configuration below.  Your Github app should be pointing to `localhost:3000` for the redirect.
 
-Also note that by default, the `bldr-run` task described in that document builds this app and starts a `web` process that serves it over port 3000. Since the dev-setup instructions below are also configured to use port 3000 (the OAuth app we use in development requires it), you could end up with a port conflict or a dev service running on a port other than 3000.
+### Repository Setup
 
-There are couple of things you can do to avoid this:
+Select a location to clone the Builder repo on your Host OS, eg, `~/Workspace` (this directory will be referred to as ${BUILDER_SRC_ROOT} in the sections below)
 
-  * If you're running `bldr-run` in a container or VM, and you've mapped port 3000 from the guest onto your local machine, remove that mapping to allow the `web` process to continue running on port 3000 on the guest, and you to run on port 3000 locally without a conflict. This is a good option of you're a regular contributor to the UI. Some providers (e.g., VirtualBox) also allow you to change this mapping in the UI without requiring a restart.
+```
+cd ${BUILDER_SRC_ROOT}
+git clone https://github.com/habitat-sh/builder.git
+```
 
-  * Set up a custom GitHub OAuth application to run the dev service somewhere other than `localhost:3000`. See below for instructions on how to do that.
+### Host OS Provisioning
 
-### Installing Node
+You will need to install `node`.
 
-We suggest using [NVM](https://github.com/creationix/nvm) (Node Version Manager) to install the version of Node specified in  [.nvmrc](.nvmrc). Follow [the instructions in the NVM docs](https://github.com/creationix/nvm#installation) to set that up.
+We suggest using [NVM](https://github.com/creationix/nvm) (Node Version Manager) to install the version of Node specified in [.nvmrc](.nvmrc). Follow [the instructions in the NVM docs](https://github.com/creationix/nvm#installation) to set that up.
```

Comment:
(I prefer `n`, but maybe that's just me)

Reply:
Not clear, can you elaborate?